packages/terra-form-input/src/FormInput.scss

```diff
@@ -0,0 +1,103 @@
+    padding-top: var(--terra-form-input-padding-top, 0.571rem);
+    transition-duration: var(--terra-form-input-transition-duration, 0s);
+    transition-property: background-size;
+    transition-timing-function: var(--terra-form-input-transition-timing-function, ease);
+    width: 100%;
+
+    &:hover {
+      border-color: var(--terra-form-input-hover-border-color, #4a4a4a);
+      border-style: var(--terra-form-input-hover-border-style, solid);
+      border-width: var(--terra-form-input-hover-border-width, 1px);
+    }
+
+    &[required] {
+      box-shadow: unset;
+    }
+
+    &[disabled] {
+      background-color: var(--terra-form-input-disabled-background-color, rgba(0, 0, 0, 0.05));
+      border-color: var(--terra-form-input-disabled-border-color, #4a4a4a);
+      border-style: var(--terra-form-input-disabled-border-style, solid);
+      border-width: var(--terra-form-input-disabled-border-width, 1px);
+      color: var(--terra-form-input-disabled-color, #000);
+      cursor: default;
+      opacity: var(--terra-form-input-disabled-opacity, 1);
+    }
+
+    &:focus {
+      background-color: var(--terra-form-input-focus-background-color, #fff);
+      border-color: var(--terra-form-input-focus-border-color, #4a4a4a);
+      border-radius: var(--terra-form-input-focus-border-radius, 0.2rem);
+      border-style: var(--terra-form-input-focus-border-style, solid);
+      border-width: var(--terra-form-input-focus-border-width, 1px);
+      opacity: var(--terra-form-input-focus-opacity, 1);
+      outline: none;
+
+      [dir=ltr] & {
+        box-shadow: var(--terra-form-input-focus-keyboard-box-shadow-ltr, 0 0 8px #a6d9f4);
+      }
+
+      [dir=rtl] & {
+        box-shadow: var(--terra-form-input-focus-keyboard-box-shadow-rtl, 0 0 8px #a6d9f4);
+      }
+    }
+  }
+
+  .form-error {
+    background-image: var(--terra-form-input-error-focus-animation-background, none);
+    border-color: var(--terra-form-input-error-border-color, #ff0b0f);
+    border-radius: var(--terra-form-input-error-border-radius, 0.2rem);
+    border-style: var(--terra-form-input-error-border-style, solid);
+    border-width: var(--terra-form-input-error-border-width, 1px);
+    opacity: var(--terra-form-input-error-opacity, 1);
+
+    &:hover {
+      border-color: var(--terra-form-input-error-hover-border-color, #ff0b0f);
+      border-style: var(--terra-form-input-error-hover-border-style, solid);
+      border-width: var(--terra-form-input-error-hover-border-width, 1px);
+    }
+
+    &[disabled] {
+      background-color: var(--terra-form-input-error-disabled-background-color, rgba(0, 0, 0, 0.05));
+      border-color: var(--terra-form-input-error-disabled-border-color, #ff0b0f);
+      border-style: var(--terra-form-input-error-disabled-border-style, solid);
+      border-width: var(--terra-form-input-error-disabled-border-width, 1px);
+      color: var(--terra-form-input-error-disabled-color, #000);
```

Comment:
needs to be updated.

Reply:
Done. 2b6d5b6031c0709535fe0bb2d78a16b7a335adc7